lib/middleware/karma.js

```diff
@@ -114,8 +114,10 @@ var createKarmaMiddleware = function (
       })
     }
 
-    // serve karma.js
-    if (requestUrl === '/karma.js') {
+    // serve karma.js, context.js, and context-debug.js
+    var jsFiles = ['/karma.js', '/context.js', '/context-debug.js']
```

Comment:
Let's drop the `/` in the filenames and it dynamically on the matches to avoid typo issues in the future.